python/tests/io/test_base.py

```diff
@@ -88,8 +95,15 @@ def exists(self):
     def to_input_file(self):
         return LocalInputFile(location=self.location)
 
-    def create(self, overwrite: bool = False) -> None:
-        return open(self.parsed_location.path, "wb" if overwrite else "xb")
+    def create(self, overwrite: bool = False) -> OutputStream:
+        output_file = open(self.parsed_location.path, "wb" if overwrite else "xb")
+        if not isinstance(output_file, OutputStream):
+            raise RuntimeError(
+                """
```

Comment:
nit: the formatting on this looks weird, is there some way to get black to make a better choice?

Reply:
We've been planning on increasing the auto formatter's line length setting for a while so I took this opportunity to open PR #4025 length for that.

Reply:
Merged the line length update.